### PR TITLE
add missing liscense info of gumby

### PR DESCRIPTION
**Pull request by limitimil**
note: https://github.com/GumbyFramework/Gumby

### CODE REVIEW

File: ajax/libs/gumby/package.json [hunk 1]
```diff
@@ -18,5 +18,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/GumbyFramework/Gumby.git"
-  }
+  },
+  "license":"MIT"
+
```
Comment (Amomo) [inline ajax/libs/gumby/package.json]: unnecessary blank

**Comment by Amomo** [comment]
Hi @limitimil 
As cdnjs has many commits in repository, we expect you use one commit to finish this work.
Could you squash your commits into one commit?

You can close this issue and open a new one, or ask us if you need any help! :-)

File: ajax/libs/gumby/package.json [hunk 2]
```diff
@@ -18,5 +18,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/GumbyFramework/Gumby.git"
-  }
+  },
+  "license":"MIT"
```
Comment (Amomo) [inline ajax/libs/gumby/package.json]: lost a blank before `"MIT"`

**Comment by Amomo** [comment]
Hi @limitimil 
now you have 3 commits in this PR, we expect you use `one commit` to finish this work.

Since you can't modify the commit message or delete your old commit in your browser.
You need to create a new branch to send a new PR.

Or you can do it locally, clone your fork cdnjs repo, and pull your branch "limitimil-gumby-license" to local, so you can modify the commit message by git commands.

**Comment by limitimil** [comment]
Thanks, I will make another branch.

**Pull request closed without merging**